.github/workflows/rust.yml

```diff
@@ -277,7 +277,9 @@ jobs:
           rustup default stable
           rustup component add rustfmt
       - name: Run
-        run: ci/scripts/rust_fmt.sh
+        run: |
+          echo '' > datafusion/proto/src/generated/datafusion.rs
```

Comment:
I think this is a left over from https://github.com/apache/arrow-datafusion/pull/3333 -- but I also don't think it hurts to leave it in this PR